examples/resources/snowflake_secret_with_client_credentials/resource.tf

```diff
@@ -0,0 +1,9 @@
+# basic resource
+resource "snowflake_secret_with_client_credentials" "test" {
+  name               = "EXAMPLE_SECRET"
+  database           = "EXAMPLE_DB"
+  schema             = "EXAMPLE_SCHEMA"
+  api_authentication = "EXAMPLE_SECURITY_INTEGRATION_NAME"
```

Comment:
We have the `fully_qualified_name` field in most of the resources, and we prefer using this instead. Security integrations have simple IDs, but schema-level objects and resources with arguments have complex IDs that are error-prone. Just for the future :)